resources/languages/ko.json

```diff
@@ -2,211 +2,211 @@
+				"success": "견본 파일을 %s에 성공적으로 저장했습니다."
 			},
 			"schematic": {
-				"missing-name": "설계도에 유효한 이름이 지정되지 않았습니다.",
-				"success": "%s에 대한 설계도 파일을 저장했습니다."
+				"missing-name": "개략도에 올바른 이름이 입력되지 않았습니다.",
+				"success": "개략도 파일을 %s에 성공적으로 저장했습니다."
 			}
 		},
 		"paste": {
-			"description": "이전에 복제 된 영역을 세계에 붙여 넣습니다.",
+			"description": "이전에 복제한 영역을 세계에 붙여넣습니다.",
 			"copy": {
-				"no-copies": "붙여 넣기 할 수있는 사본이 없습니다.",
-				"success": "보았던 위치에 사본을 붙여 넣었습니다."
+				"no-copies": "붙여넣을 복사본을 찾을 수 없습니다.",
+				"success": "보았던 위치에 복사본을 성공적으로 붙여넣었습니다."
 			},
 			"template": {
-				"nonexistent": "%s 템플릿을 템플릿 디렉토리에서 찾을 수 없습니다.",
-				"success": "보았던 위치에 %s 템플릿을 붙여 넣었습니다."
+				"nonexistent": "견본 디렉터리에서 %s 견본을 찾을 수 없습니다.",
+				"success": "보았던 위치에 %s 견본을 성공적으로 붙여넣었습니다."
 			},
 			"schematic": {
-				"nonexistent": "디렉토리에서%s을(를) 찾을 수 없습니다.",
-				"success": "위치에 있는%s의 회로도를 성공적으로 붙였습니다."
+				"nonexistent": "개략도 디렉터리에서 %s 개략도를 찾을 수 없습니다.",
+				"success": "보았던 위치에 %s 개략도를 성공적으로 붙여넣었습니다."
 			}
 		}
 	},
 	"ui": {
 		"preset-menu": {
 			"title": "사전 설정 메뉴",
-			"subtitle": "계속하려면 버튼을 클릭하십시오.",
-			"create": "생성",
+			"subtitle": "계속하려면 버튼을 클릭하세요",
+			"create": "만들기",
 			"delete": "삭제",
 			"select": "선택",
 			"list": "목록",
 			"exit": "나가기"
 		},
 		"preset-edit": {
 			"title": "사전 설정 편집 메뉴",
-			"name": "프리셋 이름",
+			"name": "사전 설정 이름",
 			"size": "브러시 크기",
-			"shape": "브러시 모양",
-			"type": "브러시 타입",
-			"hollow": "할로우 브러쉬",
-			"decrement": "브러쉬 제거",
+			"shape": "브러시 도형",
+			"type": "브러시 종류",
+			"hollow": "빈 브러시",
+			"decrement": "감소 브러시",
 			"height": "브러시 높이",
-			"perfect": "브러시 모양 마무리",
-			"blocks": "브러시 블럭",
+			"perfect": "브러시 도형 완벽",
+			"blocks": "브러시 블록",
 			"obsolete": "쓸모없는 블록",
-			"biome": "브러시 바이옴",
+			"biome": "브러시 생물군계",
 			"tree": "브러시 나무"
 		},
 		"preset-selection": {
 			"title": "사전 설정 선택 메뉴",
-			"subtitle": "미리 설정을 적용하려면 버튼을 클릭하십시오."
+			"subtitle": "사전 설정을 적용하려면 버튼을 클릭하세요"
 		},
 		"preset-deletion": {
-			"title": "프리셋 삭제 메뉴",
-			"subtitle": "미리 설정을 삭제하려면 버튼을 클릭하십시오."
+			"title": "사전 설정 삭제 메뉴",
+			"subtitle": "사전 설정을 삭제하려면 버튼을 클릭하세요"
 		},
 		"preset-list": {
 			"title": "사전 설정 목록 메뉴",
-			"subtitle": "미리보기를 편집 /보기하려면 버튼을 클릭하십시오."
+			"subtitle": "사전 설정을 편집/보려면 버튼을 클릭하세요"
 		},
 		"preset-creation": {
-			"title": "프리셋 생성 메뉴",
-			"name": "프리셋 이름",
+			"title": "사전 설정 만들기 메뉴",
+			"name": "사전 설정 이름",
 			"size": "브러시 크기",
-			"shape": "브러시 모양",
-			"type": "브러시 타입",
-			"hollow": "할로우 브러쉬",
-			"decrement": "브러쉬 제거",
+			"shape": "브러시 도형",
+			"type": "브러시 종류",
+			"hollow": "빈 브러시",
+			"decrement": "감소 브러시",
 			"height": "브러시 높이",
-			"perfect": "브러시 모양 마무리",
-			"blocks": "브러시 블럭",
-			"obsolete": "O쓸모없는 블록",
-			"biome": "쓸모없는 블록",
+			"perfect": "브러시 도형 완벽",
+			"blocks": "브러시 블록",
+			"obsolete": "쓸모없는 블록",
+			"biome": "브러시 생물군계",
 			"tree": "브러시 나무"
 		},
 		"main-menu": {
 			"title": "BlockSniper 메뉴",
-			"subtitle": "계속하려면 버튼을 클릭하십시오.",
+			"subtitle": "계속하려면 버튼을 클릭하세요.",
```

Comment:
(English) Is this inconsistency intended? Others don't have a full stop, but this one has it.

Reply:
Ah, it's fine like this, I'll go and cleanup the dots in a next commit. Thanks for noticing!